homeassistant/components/youtube/__init__.py

```diff
@@ -52,3 +56,17 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     if unload_ok := await hass.config_entries.async_unload_platforms(entry, PLATFORMS):
         hass.data[DOMAIN].pop(entry.entry_id)
     return unload_ok
+
+
+async def delete_devices(
+    hass: HomeAssistant, entry: ConfigEntry, coordinator: YouTubeDataUpdateCoordinator
+) -> None:
+    """Delete all devices created by integration."""
```

Comment:
The docstring is confusing compared to the PR title.

Reply:
I mean, now it's true 😂